app/Http/Controllers/UserListController.php

```diff
@@ -39,23 +39,23 @@ public function delete_game($listid, $itemid){
         return \Redirect::back();
     }
 
-    public function delete($listid){
-        if(\Auth::check()){
-            if(\Auth::user()->hasRole('admin')){
+    public function delete($listid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->